src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -78,7 +81,7 @@ public class LogManagerService extends PluginService {
     public static final String LOGS_UPLOADER_PERIODIC_UPDATE_INTERVAL_SEC = "periodicUploadIntervalSec";
     public static final String LOGS_UPLOADER_CONFIGURATION_TOPIC = "logsUploaderConfiguration";
     public static final String SYSTEM_LOGS_COMPONENT_NAME = "System";
-    public static final String PERSISTED_CURRENT_PROCESSING_FILE_NAME = "currentProcessingFileName";
+    public static final String PERSISTED_CURRENT_PROCESSING_FILE_HASH = "currentProcessingFileHash";
```

Comment:
What would happen when customer upgrades log manager from an older version?
1. Would the new version upload logs that's already uploaded by an older version?
2. How does log manager clean up the deprecated config nodes?

Reply:
Thanks for reminding. This will be handled in the next PR.

Reply:
I added this on another comment but I think we should keep this the old variable and still write to it just stop reading from it and read from the hash to avoid any potential breaking change add a @depreacted annotation and a comment to remove it later, that will derisk things in case of upgrade/downgrades